test/Performance/benchmark/sqlsrv/SqlsrvUpdateBench.php

```diff
@@ -45,7 +46,7 @@ public function generateUpdateParams(){
      * Note that, every update calls prepare, bindParam and execute APIs.
      */
     public function benchUpdateWithPrepare(){
-        for( $i=0; $i<1000; $i++ ){
+        for( $i=0; $i<100; $i++ ){
```

Comment:
No. Since every tests have its own specific iterations/loops per iteration, I decided to make the number of iterations test specific.